src/pages/eventdetails.js

```diff
@@ -1,5 +1,21 @@
-import './eventdetails.css'
+import EventCard from "../components/EventCard";
+import "./eventdetails.css";
 
-export default function EventDetails(){
-    return (<h1>Event Details</h1>)
+const dummyData = {
```

Comment:
Can this be moved into a separate data file for consistency and to keep this file clean?

Reply:
See commit [6b3dc41](https://github.com/acm-uic/flourish-2023/pull/75/commits/6b3dc41a95f404d828a750c8ef02f51d0298a298)